tests/topotests/lib/pim.py

```diff
@@ -966,7 +954,7 @@ def verify_join_state_and_timer(tgen, dut, iif, src_address, group_addresses, ex
     return True
 
 
-@retry(retry_timeout=80)
+@retry(retry_timeout=120)
```

Comment:
this is the timeout change you mentioned in the weekly call, right?

Reply:
yes